press-this.php

```diff
@@ -686,7 +686,7 @@ public function serve_app_html() {
 	<div id="wppt_adminbar" class="adminbar">
 		<h1 id="wppt_current_site" class="current-site">
 			<span class="dashicons dashicons-wordpress"></span>
-			<a href="<?php echo esc_url( home_url( '/' ) ); ?>" target="_blank"><?php bloginfo( 'name' ); ?></a>
+			<a href="<?php echo esc_url( home_url( '/' ) ); ?>" target="_blank" tabindex="-1"><?php bloginfo( 'name' ); ?></a>
```

Comment:
We should remove this from being in tab order by making it a span instead of an anchor.